sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -74,7 +86,36 @@ internal class RadarApiClient(
         )
     }
 
-    internal fun getConfig() {
+    data class RadarMeta(
```

Comment:
I'd like to start getting away from all these internal inner classes - can you move this class to the `io.radar.sdk.model` directory and mark it as `internal`?